libc/test/src/math/smoke/CMakeLists.txt

```diff
@@ -155,6 +155,34 @@ add_fp_unittest(
 
 )
 
+add_fp_unittest(
+  faddl_test
+  NEED_MPFR
+  SUITE
+    libc-math-unittests
+  SRCS
+    faddl_test.cpp
+  HDRS
+    AddTest.h
+  DEPENDS
+    libc.src.math.faddl
+    libc.src.__support.FPUtil.basic_operations
+)
```

Comment:
74a1ca504bf60f02431140ee72dbe1c158556237 removed the calls to `fputil::getpayload` from libc/test/src/math/smoke/AddTest.h, but didn't remove the BasicOperations.h include. Could you remove it and update the dependencies of smoke tests that use AddTest.h?

Reply:
update it with add_sub?

Reply:
No, it doesn't use `fputil::generic::add_sub` directly, either. It included BasicOperations.h to use `fputil::getpayload`, but use of that has since been removed.

I use Neovim with Clangd, and I get a warning saying "Included header BasicOperations.h is not used directly". I get the same for errno_macros.h, fenv_macros.h, and Test.h, but these are proxy headers, and we do use them as they are intended to be.

Reply:
Just remove the `#include ".../BasicOperations.h"` lines and the `libc.src.__support.FPUtil.basic_operations` target from these tests' dependencies, basically.

Reply:
let me know if that is good